### PR TITLE
Card Pre-title

### PR DESCRIPTION
**Pull request by joewhitsitt**
Screenshot 2024-11-01 at 11 22 31 AM

easy peasy?!

# Test

- `nvm use`
- `yarn install`
- `yarn storybook`
- Go to the card component and see the pretitle story. See the pinned icon and text.
- Check the code for proper naming conventions and such?

### CODE REVIEW

File: src/scss/components/card.scss [hunk 1]
```diff
@@ -229,6 +229,13 @@
 
 // === End: Media === //
 
+// === Pre-Title === //
+.card__pre-title {
+  font-size: 0.75rem;
+  color: $med-gray;
```
Comment (bspeare) [inline src/scss/components/card.scss:235]:
```suggestion
  color: $med-gray;
   i,
  span[class*="fa-"],
  svg {
    margin-right: .2rem;
  }
```

**Review by bspeare — DISMISSED** [review]
Looks good!  I wonder if we can move the pre_title above the title in the controls? Should the label be "Pre-title" which is our label for the banner in SiteNow? 

Screenshot 2024-11-01 at 11 24 55 AM

**Comment by joewhitsitt** [comment]
I don't know if controls can be sorted. I am not finding an easy way. i see docs about category sorting and alpha or required-first but not custom weight ordering.

**Comment by joewhitsitt** [comment]
maybe i am just missing something obvious since it looks like the card content category is sorted otherwise

Update: figured it out. https://github.com/uiowa/uids/pull/940/commits/1ab78e1ec5a360b687614e98856c2ea6e7efe685

**Review by bspeare — APPROVED** [review]
Changes look good, thanks!

**Pull request merged**